### PR TITLE
feat: Improve the UI of projects list `main page` and `projects page`.

### LINKED ISSUE #7: Improve projects UI
Currently projects are cards at the bottom of the page. It looks all too similar to the other cards on the page. 

A few suggestions:

- Add an image to the project preview.
- Make the cards bigger and display more information on them, like who built it, and what not
- Improve the project page when you click on it, so it's easy to navigate back to the main site

### PR DESCRIPTION
**Pull request by KhudaDad414**
## Context
The project's list on the web site's main page was a simple card that is not visually pleasing to the eye. 

## Implementation
- A background image was added to each card.
- Using a white gradient, we are making the text more readable.
- On hover is added to the list items which changes the visuals.
- An extra field is added to the project so we can express the tech stack that the project was developed with.

## Screenshots

Screenshot 2022-01-12 183419

## Authors
@riyadevvarshney11 and @KhudaDad414 

## Checklist
- [x] My Pod Leader knows I'm working on this Pull Request
- [x] I've explained what the Pull Request is adding.
- [x] I've explained why this is important.

## Related Issue
Resolves #7

### CODE REVIEW

**Comment by yyjlincoln** [comment]
Resolved merge conflicts so you will be able to merge more easily when the PR is ready to merge.

**Comment by yyjlincoln** [comment]
Nice improvements! Can't wait to see when its PR is ready to go!

**Comment by yyjlincoln** [comment]
Resolved merge conflicts

**Comment by yyjlincoln** [comment]
Lots of code style changes (number of spaces etc) will increase the likelihood of getting merge conflicts. I'm not quite sure if that's the new `.editorconfig` setting but if possible, minimise style changes (i.e. dont change the line just to adjust its indentation) so we don't get too many merge conflicts.

**Review by VaibhaveS — COMMENTED** [review]
Hey it looks super cool!

image

However in the project page, in certain areas the text also darkens during dark mode, not a big deal though

**Comment by yyjlincoln** [comment]
> @VaibhaveS: Hey it looks super cool!
> 
> image
> 
> However in the project page, in certain areas the text also darkens during dark mode, not a big deal though

That's now fixed. I've also reformatted the code according to `.editorconfig` & fixed the dark mode feature for the project card. Can you review the code again & give approval if you think it's ready to go?

File: _includes/projects.html [hunk 1]
```diff
@@ -3,17 +3,20 @@ <h2>Projects</h2>
...
-        <div class="info">
-          <p class="title">{{ item.title }}</p>
-          <p class="location">{{ item.event }}</p>
-          <p class="dates">{{ item.date }}</p>
-        </div>
-      </div>
-    </a>
-    {% endfor %}
-  </div>
-
```
Comment (yyjlincoln) [inline _includes/projects.html:19]: This has been redesigned and nobody is touching this file so I formatted the code according to `.editorconfig` to ensure consistent code styles. We shouldn't worry about this formatting change.

File: _layouts/default.html [hunk 2]
```diff
@@ -15,4 +15,6 @@
 
 {% include footer.html %}
 
-</html>
\ No newline at end of file
+{% include toggle.html %}
+<!-- Global Wrapper, excluding index.html -->
+</html>
```
Comment (yyjlincoln) [inline _layouts/default.html:20]: I also changed this file (`darkmode`) as I made it and it's a really small fix so I didn't bother to start another branch etc

**Review by yyjlincoln — APPROVED** [review]
LGTM! I also made some code formatting changes and we shouldn't worry about those changes. It's unlikely for them to break anything. Great work!

**Review by anubhav06 — APPROVED** [review]
LGTM! 🎉

**Comment by yyjlincoln** [comment]
Merging now

**Pull request merged**